tests/resources/dsp-operator/test-dspo-cr.yaml

```diff
@@ -19,36 +19,13 @@ spec:
     image: quay.io/modh/odh-ml-pipelines-viewercontroller-container:v1.18.0-8
   mlpipelineUI:
     image: quay.io/opendatahub/odh-ml-pipelines-frontend-container:beta-ui
-    configMap: ds-pipeline-ui-configmap
+    configMap: ds-pipeline-ui-configmap-sample
```

Comment:
don't need this field, that's basically the default configmap

Reply:
Made the required changes.
Ready for review again @HumairAK 
thanks :smile: